_pages/install/integrations/cds-integration.md

```diff
@@ -0,0 +1,239 @@
+
+    ```ts
+    import { CdsModule } from '@spartacus/cds';
+    ```
+
+1. Add the `CdsModule` to `app.module.ts`. 
+
+    The following is an example:
+
+    ```ts
+    @NgModule({
+      imports: [
+        CdsModule.forRoot({
+          cds: {
+            tenant: 'my-tenant',
+            baseUrl: 'https://api.us.context.cloud.sap',
+            endpoints: {
+              strategyProducts: '/strategy/${tenant}/strategies/${strategyId}/products',
+            },
+            profileTag: {
+              javascriptUrl: 'https://tag.static.us.context.cloud.sap/js/profile-tag.js',
+              configUrl:
+                'https://tag.static.stage.context.cloud.sap/config/my-config123',
+            },
+          },
+        }),
+        ...
+    ```
+    The following is a summary of the parameters of the `CdsModule`:
+
+    - **tenant:** Set this to your testing or production tenant, as required. For more information, see [Tenant Provisioning](https://help.sap.com/viewer/49cbe38e90db448eae460907a97d177b/SHIP/en-US).
+    - **baseUrl:** Replace the value shown in the example with the base URL of your Context-Driven Services environment.
+    - **strategyProducts:** Set this value as shown in the example.
+    - **javascriptUrl:** Specify the URL of the Profile Tag version you wish to use. It is recommended that you use the URL for the latest version of Profile Tag (for example, `http://tag.static.us.context.cloud.sap/js/profile-tag.js`). For more information, see [Deciding Which Profile Tag Link to Use](https://help.sap.com/viewer/8a676837bd4245f69cc8a2fbdae2b945/SHIP/en-US/2f49c91ca16344de951921e1be50c025.html) on the SAP Help Portal.
+    - **configUrl:** Specify the URL of the Profile Tag configuration that you have created in Context-Driven Services. For more information, see [Profile Tag Overview](https://help.sap.com/viewer/8a676837bd4245f69cc8a2fbdae2b945/SHIP/en-US/44cb2bd7706a48c6a3b915078d2c384d.html) on the SAP Help Portal.
+    - **allowInsecureCookies:** This is an optional parameter (not show in the example above) that specifies whether Profile Tag should set insecure cookies. The default value is `false`. If you are running on HTTP, set this parameter to `true`. For example, if you are using a local back end, `allowInsecureCookies` must be set to `true`. In production, it should always be set to `false`.
+    - **gtmId:** This is an optional parameter (not show in the example above) that is used to integrate Profile Tag with Google Tag Manager. For more information, see [Profile Tag](https://help.sap.com/viewer/8a676837bd4245f69cc8a2fbdae2b945/SHIP/en-US/3bccaa4bd20441fd88dcfc1ade648591.html) on the SAP Help Portal.
+
+1. Configure your `environment.ts` file by providing your `occBaseUrl`. The following example shows the `occBaseUrl` set to a locally-installed back end:
+
+    ```ts
+    export const environment = {
```

Comment:
I guess this isn't unique to the cds-integration, but I will leave it up to you to decide if this is worth including in this section ;)

Reply:
Ah, I see. This step was included in the original integration doc, but in my own instance, I just tried commenting it out and the app still compiled without complaining. 
Also, I did a quick scan of the spartacus docs, and I see that adding `occBaseUrl` is documented elsewhere, particularly for contributers... So really, maybe it's not needed here. 
@corey-aloia , if you think it's safe to remove this step, I think I will, since, as you say, it's not unique to CDS.

Reply:
yea, I would say to remove it

Reply:
Done.